src/no/runsafe/warpdrive/database/WarpRepository.java

```diff
@@ -138,19 +127,27 @@ private String cacheKey(IPlayer creator, String name, boolean publicWarp)
 		return String.format("%s:%s", creatorName, name);
 	}
 
+	/**

```

Comment:
Maybe just rewrite this to accept a `null` for `owner` to fetch public warps? I probably had an thought of adding public warps for players when I designed this API, but that is not a planned feature to my knowledge.